Naming/src/main/java/com/epam/engx/cleancode/naming/task5/FileExtensionsPredicate.java

```diff
@@ -3,17 +3,17 @@
 
 import com.epam.engx.cleancode.naming.task5.thirdpartyjar.Predicate;
 
-public class FileExtPred implements Predicate<String> {
+public class FileExtensionsPredicate implements Predicate<String> {
 
-    private String[] extns;
+    private String[] extensions;
 
-    FileExtPred(String[] extns) {
-        this.extns = extns;
+    FileExtensionsPredicate(String[] extensions) {
+        this.extensions = extensions;
     }
 
     @Override
     public boolean test(String fileName) {
-        for (String extension : extns) {
+        for (String extension : extensions) {
```

Comment:
```suggestion
        for (String allowedExtension : allowedExtensions) {
```
I believe that will help to clearly understand what is going here.